doc/releases/migration-guide-3.6.rst

```diff
@@ -356,6 +356,9 @@ Device Drivers and Device Tree
 
   (:github:`66427`)
 
+* The property ``in-gpios`` defined in :dtcompatible:`worldsemi,ws2812-gpio` has been
+  renamed to ``gpios``.
+
```

Comment:
This needs to go into the migration guide for v3.7.0 instead.

Reply:
Thank you for notifying.
I moved it to `migration-guide-3.7.rst`.